.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{js,json,jsonld,yaml,yml}]
+indent_style = space
+indent_size = 2
```

Comment:
This looks like it should be git ignored.

Reply:
No, we commit these other places: https://github.com/digitalbazaar/jsonld-cli/blob/main/.editorconfig

They're super useful for consistent line endings, final new lines, and indentation preferences. It should not be ignored, because it is not idiosyncratic and is meant to be shared by design. 😃

Reply:
Ok.